llvm/include/llvm/CodeGen/SelectionDAGISel.h

```diff
@@ -61,6 +61,10 @@ class SelectionDAGISel : public MachineFunctionPass {
   /// Used to report things like combines and FastISel failures.
   std::unique_ptr<OptimizationRemarkEmitter> ORE;
 
+  /// True if the function currently processing is in the function printing list
+  /// (i.e. `-filter-print-funcs`).
```

Comment:
Probably worth noting here (or in the macro?) that we want to cache this value so we don't have to query the module and scan the list at every print? seems like a detail that would be easy to miss later.

Reply:
Done.